.changeset/many-roses-hammer.md

```diff
@@ -0,0 +1,89 @@
+---
+"@primer/react": patch
+---
+
+## Button deprecation
+
+The new set of Button components brings a much needed update.
+The older version was a set of seven different components. Each variant of the component had a separate component.
+The guidelines to put in icons inside the buttons was also vague. With the new components, we now have a great guidance for common button usage.
+
+## Change list
+
+### Button variants
+
+We now support a variant property which takes values `primary`, `invisible`, `outline` and `danger`
+
+#### Before
+
+```jsx
```

Comment:
Todo for later.. we'd want to make these tables, but we can do this in a follow up PR